flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
why? It's Java.

Reply:
essential dependency jnr-fuse doesn't support aarch64 yet. we're working on an alternative and add further arches later.